js/slider.js

```diff
@@ -0,0 +1,20 @@
+const slides = document.getElementsByClassName("mySlides");
+let slideIndex = 0;
+for (let i = 0; i < slides.length; i++) {
```

Comment:
I guess, this loop is not required, since `display: none` is already set in the css.